### DISCUSSION TITLE
Issue with Bare Metal deployment

### DISCUSSION THREAD

**Discussion by anitha-gadde** [open-metadata/OpenMetadata#8020] [Q&A] [answered]
I have done all the steps but unable to open open metadata ui: http://localhost:8585/..please suggest .

**Comment by pmbrull** [comment, marked as answer]
hi @anitha-gadde, I'd recommend reaching out to in Slack https://slack.open-metadata.org/ and sharing some further details on how are you setting this up, version, logs, etc. Thanks!
